kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -24,10 +24,17 @@ const publicMethods = [
  * is configured in webpack.config.js
  * @classdesc
  */
-function Lib() {
+function Lib(kolibriGlobal) {
   // libraries
   this.loglevel = require('loglevel');
   this.vue = require('vue');
+  this.vue.mixin({
+    methods: {
+      url: (url_name, ...args) => {
+        return kolibriGlobal.urls[url_name](...args);
+      }
```

Comment:
I see - that's convenient.

For API calls, I'd imagine we wouldn't need a vue mixin because those will be made in vuex actions, outside of vue components - does that seem right?